### PR TITLE
Update README.md with improved content

### PR DESCRIPTION
**Pull request by relvinarsenio**
Refined the content of the README.md file to provide clearer instructions for uninstalling Spotify on Windows. Added usage instructions, script details, and a disclaimer. Also, included a link to the batch script. This commit enhances the usability and transparency of the provided information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,26 @@
...
-Just download and run [Uninstall-Spotify.bat](https://raw.githack.com/amd64fox/Uninstall-Spotify/main/Uninstall-Spotify.bat)
+If you're looking to uninstall Spotify from your Windows system, you can use the provided batch script.
+
+## Usage
+
+1. Download the batch script: [Uninstall-Spotify.bat](https://raw.githack.com/amd64fox/Uninstall-Spotify/main/Uninstall-Spotify.bat).
+2. Double-click the downloaded `Uninstall-Spotify.bat` file to run it.
+3. Follow any on-screen prompts or instructions that may appear.
+
+Please note that running batch scripts from the internet can be risky. Ensure you trust the source before executing any script on your system.
```
Comment (amd64fox) [inline README.md]: since you have already written a disclaimer below, this line is not needed

File: README.md [hunk 2]
```diff
@@ -1,3 +1,26 @@
 # Uninstall Spotify for Windows
 
-Just download and run [Uninstall-Spotify.bat](https://raw.githack.com/amd64fox/Uninstall-Spotify/main/Uninstall-Spotify.bat)
+If you're looking to uninstall Spotify from your Windows system, you can use the provided batch script.
+
+## Usage
+
+1. Download the batch script: [Uninstall-Spotify.bat](https://raw.githack.com/amd64fox/Uninstall-Spotify/main/Uninstall-Spotify.bat).
+2. Double-click the downloaded `Uninstall-Spotify.bat` file to run it.
+3. Follow any on-screen prompts or instructions that may appear.
```
Comment (amd64fox) [inline README.md]: delete this line
upon execution, the script does not provide on-screen prompts or instructions;

**Comment by amd64fox** [comment]
Also, add somewhere that the script only deletes the desktop version.

**Comment by relvinarsenio** [comment]
I have already changed it, Is it like that?

**Comment by amd64fox** [comment]
almost

**Comment by amd64fox** [comment]
now it is better
Thanks

**Pull request merged**